ShenkinStore/Controllers/UsersController.cs

```diff
@@ -167,16 +167,16 @@ public ActionResult Register(User aUser)
                 if (_context.User.Where(m => m.UserName == aUser.UserName).FirstOrDefault() == null)
                 {
                     User objuser = new User();
-                    objuser = aUser;
                     objuser.CreatedOn = DateTime.Now;
                     objuser.Email = aUser.Email;
                     objuser.UserName = aUser.UserName;
                     objuser.Password = aUser.Password;
                     objuser.Phone = aUser.Phone;
-                    objuser.SuccessMessege = "User is successfully add.";
                     _context.User.Add(objuser);
                     _context.SaveChanges();
-                    return View("Login");
+                    objuser = new User();
```

Comment:
Why create a new object? this will override the first assignment.

Reply:
It meant to return an empty form with the success-message to the user. i will remove it